vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/order_handler.rb

```diff
@@ -15,10 +15,8 @@ def initialize(occupancy)
       end
 
       def process
-        return unless occupancy.account_id?
-
         find_or_create_order
-        fulfill_order if occupancy.complete?
+        occupancy.attempt_order_complete!
```

Comment:
yeah.. this is a little odd since the code was sorta shared because of the orphaned case... perhaps if I move order completion out of `Occupancy` and just have a `#complete_order`-type method on both `OrderHandler` and `OccupancyHandler`?